hadoop-ozone/dist/src/main/smoketest/recon/recon-api.robot

```diff
@@ -136,3 +153,13 @@ Check normal api access
 
     kinit as non admin
     Check http return code      ${NON_ADMIN_API_ENDPOINT_URL}   200
+
+Check listKeys api works
+   kinit as ozone admin
+   # Create volume and bucket
+   Execute    ozone sh volume create ${VOLUME}
+   Execute    ozone sh bucket create ${VOLUME}/${BUCKET}
+   Freon OCKG    n=2    args=-s 10 -v ${VOLUME} -b ${BUCKET}
+
+   # Wait until Recon picks up the keys
+   Wait Until Keyword Succeeds     90sec      10sec        Check if the listKeys api responds OK   ${VOLUME}     ${BUCKET}
```

Comment:
Thanks a lot @siddhantsangwan for adding the test.  Unfortunately it fails with:

```
VOLUME_ALREADY_EXISTS Volume already exists
```

Instead of starting from scratch, we could add the `listKeys` request to existing test case `Check if Recon picks up OM data`, relying on the volume, bucket and keys created there.

Reply:
Thanks, I've the updated the PR with the suggested change.